<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maquette</class>
 <widget class="QDialog" name="maquette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="Gestiondesaccidents">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>891</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gesntion des accidents</string>
    </attribute>
    <widget class="QLineEdit" name="matricule">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="chauffeur">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lieu">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="fautif">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="date">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricule:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Chauffeur:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lieu:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fautif:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_Ajout">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_modifier">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>351</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="PDF">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>110</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="imprimer">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>180</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>311</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Supprimer un accident</string>
     </property>
     <widget class="QPushButton" name="PB_supp">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>matricule:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="matricule_s">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>321</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Rechecher un accident</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Matricule : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="matricule_re">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>340</y>
       <width>331</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Calculer le nombre d'accidents</string>
     </property>
     <widget class="QPushButton" name="calculer">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculer</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_calculer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>191</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Mailing</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="msg">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>501</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="email">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>400</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>receveur:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Objet:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>attachement:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mail_pass:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rec">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="objet">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pass">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="fichier">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="browse">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
